### DISCUSSION TITLE
Possible to use endnotes instead of footnotes for PDF?

### DISCUSSION THREAD

**Discussion by DamonCharlesRoberts** [quarto-dev/quarto-cli#4082] [Q&A] [answered]
I footnotes in a manuscript that I am writing. However, for some awful reason, the journal is requesting that my footnotes be replaced as endnotes. Is there a way to turn my footnotes into endnotes?

**Comment by cscheid** [comment, marked as answer]
<strike>The best solution will be for you to write a filter to catch `Note` AST elements, collect all of them, and then create a new section in the end with all of their content. It's doable, but will require some Lua: https://pandoc.org/lua-filters.html#type-note</strike>

It might be much easier than that. Take a look at this comment in a pandoc issue: https://github.com/jgm/pandoc/issues/4041#issuecomment-632965163

**Reply by DamonCharlesRoberts** [reply]
That last recommendation worked for me!

For reference for others:

I have a larger `.tex` template that I am working with and I just added: 

```
%% If need to use endnotes instead of footnotes
$if(params.endnotes)$
	\usepackage{endnotes}
	\let\footnote=\endnote
$else$

$endif$
```
to my preamble, and then at the end of the template, just added:
```
$if(params.endnotes)$
    \newpage
    \theendnotes
$endif$
```

Then in the `yaml` of my `.qmd` file, I just added:
```
---
params:
    endnotes: true
---
```
I've got a full working example here for reference: https://github.com/DamonCharlesRoberts/quarto_templates/tree/main/manuscripts

**Reply by cscheid** [reply]
This is great, thank you for expanding on my very minimal pointer!
